config/crd/bases/config.nri_balloonspolicies.yaml

```diff
@@ -106,6 +106,18 @@ spec:
                       items:
                         type: string
                       type: array
+                    preferCloseToDevices:
+                      description: 'PreferCloseToDevices: prefer creating new balloons
+                        of this type close to listed devices.'
+                      items:
+                        type: string
+                      type: array
+                    preferFarFromDevices:
```

Comment:
Could we hide `preferFarFromDevices` from CRDs? It's undocumented, untested (as an explicitly given option), and the policy fills it internally to reflect `preferCloseToDevices` configurations from other balloon types.

"Internal filling" does not mean "overwriting", but only "adding to it". In other words, it could be made part of public interface, too, but I'm not sure if we want to do that.

Reply:
I'm not sure, but I think/recall that if you set its golang struct member tag to "-" instead of `preferCloseToDevices` then it will not be available for unmarshalling (hence you shouldn't be able to set it from a CRD).